bin/windows-env.ps1

```diff
@@ -49,9 +55,8 @@ $Wix35_VERSION = '3.5.2519.20130612'
 $env:PATH = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
 if ($arch -eq 32) {
   $env:PATH = "C:\tools\mingw32\bin;" + $env:PATH
+} else {
+  $env:PATH = "C:\tools\mingw64\bin;" + $env:PATH
 }
-$env:PATH += [Environment]::GetFolderPath('ProgramFiles') + "\Git\cmd"
+$env:PATH += [Environment]::GetFolderPath('ProgramFiles') + "\Git\cmd" + ";" + "$toolsDir\$rubyPkg\bin" + ";" + "$toolsDir\$opensslPkg\bin"
 Write-Host "Updated Path to $env:PATH"
-
-# SSL root pointer.
```

Comment:
:+1: - if you could add a note about that to commit message, that would be helpful